src/osd/osd_perf_counters.cc

```diff
@@ -246,6 +246,10 @@ PerfCounters *build_osd_logger(CephContext *cct) {
   osd_plb.add_u64(
     l_osd_hb_to, "heartbeat_to_peers", "Heartbeat (ping) peers we send to");
   osd_plb.add_u64_counter(l_osd_map, "map_messages", "OSD map messages");
+  // full map definition 
+  osd_plb.add_u64_counter(l_osd_map_full, "full_map", "number of full map recieved");
```

Comment:
Let's be a little bit more verbose here.  Instead of l_osd_map_full and "full_map", how about l_osd_full_map_received and "full_map_received".  I think we can be a bit more specific with the description, something like "Number of full OSDMaps received via MOSDMap."

Same for incremental.

Reply:
Probably don't need ```// full map definition``` since the third argument is documentation of what the counter does.